### PR TITLE
add org.kde.kompare

### PR DESCRIPTION
**Pull request by Amit0617**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

**Comment by Amit0617** [comment]
bot, build org.kde.kompare

**Comment by Amit0617** [comment]
bot, build org.kde.kompare

**Comment by Amit0617** [comment]
bot, build org.kde.kompare

File: org.kde.kompare.json [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+            		"sha256": "bc64d0ae804efe2ea16b41ca2b4bf91032a3f4cdc1cd970109e260b22c748c0f" 
+            	} 
+            	]
+        },
+        {
+            "name": "kompare",
+            "buildsystem": "cmake-ninja",
+            "builddir": true,
+            "sources": [ 
+            	{
```
Comment (travier) [inline org.kde.kompare.json:33]: whitespace at the end of the line

**Review by travier — CHANGES_REQUESTED** [review]

**Comment by travier** [comment]
Can you link to upstream patch submission?

**Comment by Amit0617** [comment]
https://invent.kde.org/sdk/kompare/-/merge_requests/5

**Comment by nedrichards** [comment]
bot, build org.kde.kompare

**Comment by nedrichards** [comment]
This isn't working ideally on its own. On a Fedora 35 test system I get:

> kf.service.services: KMimeTypeTrader: MIME type "text/x-patch" not found
kf.service.services: KMimeTypeTrader: couldn't find service type "Kompare/ViewPart" 
Please ensure that the .desktop file for it is installed; then run kbuildsycoca5.
[ALSOFT] (EE) Failed to set real-time priority for thread: Operation not permitted (1)
kf.notifications: Audio notification requested, but sound file from notifyrc file was not found, aborting audio notification


and then it closes.

**Comment by Amit0617** [comment]
If there are any suggestions i will happily push those changes.

**Comment by nedrichards** [comment]
bot, build org.kde.kompare

**Comment by Amit0617** [comment]
@nedrichards does this have any problem?

**Comment by nedrichards** [comment]
Yes. I literally can't run it https://github.com/flathub/flathub/pull/2765#issuecomment-1013759627

**Comment by travier** [comment]
There is still a mix of tabs and spaces here. Look for a configuration in your editor to highlight them and fix them (spaces only).

**Comment by Amit0617** [comment]
bot, build org.kde.kompare

**Comment by travier** [comment]
This is now using tabs only but we usually use spaces only.

**Comment by travier** [comment]
Apart form that, works as expected for me.

**Comment by Amit0617** [comment]
> This is now using tabs only but we usually use spaces only.

Oops!

**Comment by travier** [comment]
LGTM. This one should be good to merge. We can merge as beta if it does not work for non-KDE setups

**Comment by nedrichards** [comment]
bot, build org.kde.kompare

**Comment by nedrichards** [comment]
bot, build org.kde.kompare

**Pull request closed without merging**